src/runner.ts

```diff
@@ -13,12 +14,12 @@ export const runRules = (
   let results: IRuleResult[] = [];
 
   for (const name in rules) {
-    if (!rules.hasOwnProperty(name)) continue;
+    if (!Object.prototype.hasOwnProperty.call(rules, name)) continue;
```

Comment:
What does this fix?

Reply:
It's not a fix, it's just a slightly safer approach as `ownHasProperty` can be overridden, in theory. Obviously it's not a case here, so I'm eager to revert the change.

Reply:
Let's revert it :).